certbot-apache/certbot_apache/tests/augeas_configurator_test.py

```diff
@@ -33,12 +33,12 @@ def test_bad_parse(self):
         self.config.parser.parse_file(os.path.join(
             self.config.parser.root, "conf-available", "bad_conf_file.conf"))
         self.assertRaises(
-            errors.PluginError, self.config.check_parsing_errors, "httpd.aug")
+            errors.PluginError, self.config.parser.check_parsing_errors, "httpd.aug")
 
     def test_bad_save(self):
         mock_save = mock.Mock()
         mock_save.side_effect = IOError
-        self.config.aug.save = mock_save
+        self.config.parser.aug.save = mock_save
```

Comment:
If we want this to be a test on the configurator, I think this should be:
```suggestion
        self.config.parser.save = mock_save
```

Reply:
Even though it tests and calls things through configurator, I moved this to parser_test as it's Augeas implementation specific and we'd end up moving it there eventually anyway. 

Also `self.config.parser.aug.save` was actually still correct because the `PluginError` is thrown by the save preflight check `ApacheParser.unsaved_files()`.